src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -276,6 +276,46 @@ type SilverlightVersion =
         | SilverlightVersion.V5 -> "v5.0"
 
 [<RequireQualifiedAccess>]
+type MonoAndroidVersion =
+    | V1
+    | V22
+    | V23
+    | V403
+    | V41
+    | V43
+    | V44
+    | V5
+    | V6
+    | V7
+    | V71
+    member this.ShortString() =
+        match this with
+        | MonoAndroidVersion.V1    -> ""
+        | MonoAndroidVersion.V22   -> "2.2"
+        | MonoAndroidVersion.V23   -> "2.3"
+        | MonoAndroidVersion.V403  -> "4.03"
```

Comment:
Compared to the XS screenshot there's also some Versions still missing. Not sure if we should add them, or just keep waiting until people complain or need them. The thing is I just took the versions paket already recognized which where based on real-world versions I found during my usage of paket on Xamarin projects. Any thoughts?